src/start/RealmScreen.js

```diff
@@ -55,7 +55,15 @@ class RealmScreen extends PureComponent<Props, State> {
 
     try {
       const serverSettings: ApiResponseServerSettings = await api.getServerSettings(realm);
-      dispatch(realmAdd(realm, new ZulipVersion(serverSettings.zulip_version)));
+      dispatch(
+        realmAdd(
+          realm,
+          serverSettings.zulip_feature_level === undefined
+            ? null
```

Comment:
See discussion above: https://github.com/zulip/zulip-mobile/pull/4079#discussion_r418365612

This can be 0 instead of null, and then the action type can have just `zulipFeatureLevel: number`.